app/routes/admin/settings/billing.js

```diff
@@ -0,0 +1,7 @@
+import Route from '@ember/routing/route';
+
+export default Route.extend({
+  model() {
+    return this.store.findAll('setting');
```

Comment:
this is already called in the index route.

Reply:
Removed the redundant store hit & used `modelFor`